kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/model/networking/TlsClusterIPClusterIngressNetworkingModel.java

```diff
@@ -61,14 +59,21 @@ public record TlsClusterIPClusterIngressNetworkingModel(KafkaProxy proxy,
 
     @Override
     public Stream<ServiceBuilder> services() {
-        String serviceName = bootstrapServiceName();
-        var bootstrapService = createService(serviceName);
-        var nodeServices = nodeIdRanges.stream()
-                .flatMapToInt(nodeIdRange -> IntStream.rangeClosed(toIntExact(nodeIdRange.getStart()), toIntExact(nodeIdRange.getEnd())))
-                .mapToObj(upstreamNodeId -> createService(bootstrapServiceName() + "-" + upstreamNodeId));
+        var bootstrapService = createService(bootstrapServiceName());
+        var nodeServices = getNodeServices();
         return Stream.concat(Stream.of(bootstrapService), nodeServices);
     }
 
+    private Stream<ServiceBuilder> getNodeServices() {
+        return nodeIdRanges.stream()
+                .flatMapToInt(nodeIdRange -> IntStream.rangeClosed(toIntExact(nodeIdRange.getStart()), toIntExact(nodeIdRange.getEnd())))
+                .mapToObj(upstreamNodeId -> createService(nodeServiceNamePrefix() + "-" + upstreamNodeId));
```

Comment:
Nit:
Move the `"-"` into the prefix function, though arguably that should just accept the nodeId as a param and include that directly.

Reply:
Oh I see you use the prefix elsewhere, but it should still include the final `-`

Reply:
have changed it to a `String suffixedServiceName(String suffix)` method instead, removes the duplication further.